content/en/docs/overview/singleton-service/singleton-service-apis.md

```diff
@@ -573,9 +573,160 @@ The value of 2 returns the formatted pattern data according to the provided lang
 8. machineTranslation: "true" or "false"
 9. checkTranslationStatus: "true" or "false"
 
+# **API - Get Version list API**
+This API is used to get a product name's all available versions
 
+#### **GET**/i18n/api/v2/translation/products/{productName}/versionlist
 
+**Parameters:**
+
+productName: **(required)** (String), to indicate the product name registered in Singleton; e.g. Testing
+
+**Return value:**
+
+(JSON Object) The request productName's all version list.
```

Comment:
it confuse me that JSON Object's is list.

Reply:
done